README.md

```diff
@@ -1,34 +1,19 @@
-![cf](https://i.imgur.com/7v5ASc8.png) lab-03-fs-readfile
-======
+# Lab 3: Node Fs Read File
 
```

Comment:
Generally, you'll want to describe how the app functions in a Read me, like how to use it.